src/Transformers/KindTransformer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Exonet\Powerdns\Transformers;
+
+class KindTransformer extends Transformer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function transform()
+    {
+        $result = (object) [
+            'kind' => $this->data['kind'],
+        ];
+        if ('Slave' == $this->data['kind']) {
```

Comment:
Please use strict compare:

```suggestion
        if ('Slave' === $this->data['kind']) {
```